manual/running_vm.md

```diff
@@ -0,0 +1,38 @@
+![Banner](./header.png)
+# How to run skiftOS on Virtual Box
+
+## Preparing the host OS
+1. Download the latest stable version of _Virtual Box_ for your host OS from [here](https://www.virtualbox.org/wiki/Downloads) or any other _similar_ service.
+2. Download the latest release `.img` file of skiftOS from [here](https://github.com/skiftOS/skift/releases/latest).
+3. Rename the extension of the downloaded `.img` file from `.img` to `.hdd`.
+
+    > **NOTE:** Incase you are using windows, file extensions may not be displayed. Read this [guide](https://www.partitionwizard.com/partitionmanager/change-file-extension-windows-10.html) to enable extensions and then rename the file.
+4. Great! You now have all the files required to run skiftOS
+
+## Creating the VM on Virtual Box
+
+5. Create a new VM by clicking on the **New** button.
+6. Give the OS a meaningful name and set **Type: Other** and **Version: Other/Unknown** and click on next.
+7. Allocate a minimum of **128MB RAM** _(512MB recommended)_ and click on next.
+8. Choose **Do not add a virtual hard disk**, click on _Create_ and ignore the warning message and press **Continue**.
+
+  ![Steps 5-8 gif](./screenshots/vm_step1.gif)
+
+## Final few Steps!
```

Comment:
I don't think this header is really needed

Reply:
okay but the gif can remain in the same place?

Reply:
Yes ;)